src/components/ContentWrapper.jsx

```diff
@@ -419,9 +419,13 @@ const ContentWrapper = (props) => {
 };
 
 ContentWrapper.propTypes = {
-  experimentId: PropTypes.string.isRequired,
+  experimentId: PropTypes.string,
```

Comment:
Removed this because it is not required (we are even handling the case where `experimentId` is undefined) and was showing up as a warning in the console